media-sound/mac/mac-4.38.recipe

```diff
@@ -0,0 +1,70 @@
+SUMMARY="A fast and powerful lossless audio compressor"
+DESCRIPTION="Monkey's Audio is a lossless audio format. This package provides \
+the mac command line utility for compressing and decompressing Monkey's Audio \
+files."
+HOMEPAGE="https://www.monkeysaudio.com/"
+COPYRIGHT="2000-2018 Matthew T. Ashland"
+LICENSE="Monkey's Audio SDK and Source Code License Agreement"
+REVISION="1"
+SOURCE_URI="https://www.monkeysaudio.com/files/MAC_SDK_438.zip"
+CHECKSUM_SHA256="1ab9a0c74a0478c3fe713908789017c53c3b333a482225d2d09990a16069f981"
+SOURCE_DIR=""
+PATCHES="mac-$portVersion.patchset"
+ADDITIONAL_FILES="Makefile"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+SECONDARY_ARCHITECTURES="x86"
+fi
+
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandBinDir=$prefix/bin
+fi
+
+libVersion="${portVersion}"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+	mac$secondaryArchSuffix = $portVersion
+	cmd:mac = $portVersion
```

Comment:
Here we have 2 solutions:
* A = we provide `cmd:mac_x86` on x86_64/x86 hybrid.
* B = we delete `$commandBinDir/mac` on x86_64/x86 hybrid.

A)
```diff
-	cmd:mac = $portVersion
+	cmd:mac$commandSuffix = $portVersion
```
B)
```diff
 PROVIDES="
 	mac$secondaryArchSuffix = $portVersion
-	cmd:mac = $portVersion
 	lib:libMAC$secondaryArchSuffix = $libVersionCompat
 	"
+if [ -z "$commandSuffix" ]; then
+PROVIDES+="
+	cmd:mac = $portVersion
+	"
+fi
 REQUIRES="
```